.circleci/config.yml

```diff
@@ -59,6 +59,7 @@ jobs:
             command: |
               pip install --user -q --upgrade pip numpy vtk
               pip install --user --progress-bar off -r requirements.txt
+              pip install --user pylsl
```

Comment:
I just realized why this "works". There is no `pylsl>=1.13` which was the original requirement. There is 1.12, which is what CircleCi installed if you [dig into the logs](https://circleci.com/gh/mne-tools/mne-python/12393). If 1.12 works then we should relax this version requirement in `README.rst` and we can add to `requirements.txt` and `environment.yml` in the `pip` section (probably).

Reply:
good eye. @larsoner would you just leave it without 1.12 or should we give it a minimum version number?

Reply:
The best thing to do would be to figure out the minimum version number needed for the realtime client code to work. But that could be annoying, so we could just say >= 1.12 in the README.rst and in our `requirements.txt` and `environment.yml`